src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -668,6 +670,9 @@ <h4 class="p2">Products that have been bought together</h4>
         </a>
       </template>
     </amp-list>
+    <!--
+      You can set the value of the `relatedProductsSrc` variable by using a button `tap` action.
```

Comment:
suggestion: "The button's tap action sets the value of the `relatedProductsSrc` to an extended list of related products."